server/src/main/java/org/apache/iotdb/db/service/IoTDB.java

```diff
@@ -169,7 +169,6 @@ public void stop() {
 
   public void shutdown() throws Exception {
     logger.info("Deactivating IoTDB...");
-    IoTDB.metaManager.clear();
```

Comment:
Why we need to delete this statement. In the clear() method, MLog and TLog will be forced to disk.

Reply:
See #2428 

Some RestartTests failed because the compaction module does not allow clear the MManamger before it is stopped.
see #2426 https://github.com/apache/iotdb/pull/2426/checks?check_run_id=1650191463 and
#2425 https://github.com/apache/iotdb/pull/2425/checks?check_run_id=1649494523